WooCommerce/build.gradle

```
@@ -372,6 +375,31 @@ task violationCommentsToGitHub(type: se.bjurr.violations.comments.github.plugin.
     ]
 }
 
+task dependencyTreeDiffCommentToGitHub(type: ViolationCommentsToGitHubTask) {
```

This is an old syntax that makes the task created eagerly. We can use the suggestion below to make the task created lazily, meaning it'll only be created if the task will run - saving some time in all other cases. You can find more information about this [in this guide](https://docs.gradle.org/current/userguide/task_configuration_avoidance.html).

```suggestion
tasks.register("dependencyTreeDiffCommentToGitHub", ViolationCommentsToGitHubTask) {
```

Please note that I realize the other `ViolationCommentsToGitHubTask` is created in the same way, but this is still new code, so I think the above is a fair suggestion. I'll also leave a suggestion for the other task if you would like to include the change for it.

---

Done